gps_to_path/launch/joy_waypoint_control.launch

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<launch>
+    <arg name="max_age" default="1.0"/>
+    <arg name="decrement_buttons" default="3,2"/>  <!-- Logitech F710: Y+X -->
+    <arg name="increment_buttons" default="3,1"/>  <!-- Logitech F710: Y+B -->
+    <arg name="step" default="1"/>
+
+    <arg name="joy_topic" default="/joy"/>
```

Comment:
As this is mostly robingas-specific, what about directly setting the default to `joy_secondary/joy`?